### PR TITLE
Removed all markdown links starting with /dotnet/api/

### PR DESCRIPTION
**Pull request by Arlodotexe**
This PR:
- Reduces all 7.x links to just the text. These links were included in the port from 7.x, and do not reflect the 8.0 prerelease. They will be added back when documentation has been published for WCT 8.0.
- Makes all markdown-based URLs absolute in the sample app. For the public documentation website, we will preprocess these and make the paths relative. This make the links usable for anyone viewing or editing the raw markdown files.

### CODE REVIEW

File: components/RichSuggestBox/samples/RichSuggestBox.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ RichSuggestBox resembles text controls commonly found in social applications whe
 
 ## Remarks
 
-When a suggestion is selected, `RichSuggestBox` assigns the selected item a unique [Guid](/dotnet/api/system.guid) and a display text (provided by the developer) to make up a token. The display text is then padded with [Zero Width Space](https://unicode-table.com/200B/)s (ZWSP) and inserted into the document as a hyperlink using the identifier as the link address. These hyperlinks are tracked and validated on every text change.
```
Comment (michael-hawker) [inline components/RichSuggestBox/samples/RichSuggestBox.md:32]: We'll only want to modify the toolkit ones.

**Review by michael-hawker — DISMISSED** [review]

File: components/Animations/samples/Animations.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ issue-id: 0
 icon: Assets/ImplicitAnimations.png
 ---
 
-The [`ImplicitAnimationSet`](/dotnet/api/microsoft.toolkit.uwp.ui.animations.ImplicitAnimationSet) type is the equivalent of [`AnimationSet`](/dotnet/api/microsoft.toolkit.uwp.ui.animations.AnimationSet) in the context of implicit [Composition animations](/windows/uwp/composition/composition-animation). It represents a set of implicit animations that can only run on the [Composition layer](/windows/uwp/composition/visual-layer) and that are available in three categories: show, hide, and implicit animations. `ImplicitAnimationSet` restricts the type of contained animations to objects implementing the [`IImplicitTimeline`](/dotnet/api/microsoft.toolkit.uwp.ui.animations.IImplicitTimeline) interface to offer an additional level of build-time safety when constructing animations from XAML. Similar to the other interfaces used for explicit animations this architecture is also extensible in that users can also easily plug in their custom types implementing this interface into an `ImplicitAnimationSet` collection.
```
Comment (michael-hawker) [inline components/Animations/samples/Animations.md:15]: The composition animations link to the /windows/uwp api was stripped, but should remain.

Comment (Arlodotexe) [reply components/Animations/samples/Animations.md:15]: Good catch, let me adjust and try again

Comment (Arlodotexe) [reply components/Animations/samples/Animations.md:15]: Fixed. I've also made sure that all link text left behind is wrapped in a `CodeBlock` if there wasn't one, for readability and consistency.

**Review by michael-hawker — APPROVED** [review]

**Pull request merged**